src/styles/theme/theme.js

```diff
@@ -4,13 +4,13 @@ import GlobalStyles from "../global.styles";
 function theme() {
   return {
     fonts: {
-      roboto: "Roboto, sans-serif",
-      condensed: "Roboto Condensed, sans-serif",
+      inter: "Inter, sans-serif",
     },
     colors: {
       white: "#FFFFFF",
-      black: "#000000;",
```

Comment:
dont remove this and instead use dark for the other one cause this what i did in the nav component